scripts/kata-install/osc-kata-install.sh

```
@@ -154,22 +153,48 @@ install_kata() {
 		echo "Running in chroot: $install_cmd"
 		chroot /host bash -c "$install_cmd"
 
+		# rpm-ostree install --apply-live updates /usr immediately but
+		# /etc changes from RPMs only land after reboot
+		# (OSTree three-way merge).
+		# Copy the /etc files we find in the rpms.
+		# This is notably necessary for CRI-O drop-ins so that the CRI-O
+		# restart below is effective
+		for rpm_path in "${install_rpms[@]}"; do
+		    tar cf - $(rpm -qpl $rpm_path | grep "^/etc/") |
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)